### DISCUSSION TITLE
can Commonhaus help Jackson with new logo for a major version update?

### DISCUSSION THREAD

**Discussion by JooHyukKim** [commonhaus/foundation#202] [General]
This is [offshoot from discussion in main Jackson-repo.](https://github.com/FasterXML/jackson/discussions/230#discussioncomment-10873538)

So the idea/question is this.
Can Commonhaus help its family members with designing tasks like creating a new logo?

We started to prepare a new logo to go together with Jackson's major version update -- from version 2 to version 3.
The logo is outdated due to long history of Jackson being around for more than 10 years.

So yeah, let me how creative we can get :)))

cc. @cowtowncoder

**Comment by ebullient** [comment]
Sorry for being quiet on this one. There are things we need to get done first. Short answer is, yes. I would like to be able to help with this. @evanchooly is also looking for a new logo for Morphia.
